analyzers/tests/SonarAnalyzer.UnitTest/ResourceTests/GeneratedResourcesTest.cs

```diff
@@ -1,124 +0,0 @@
-﻿/*
- * SonarAnalyzer for .NET
- * Copyright (C) 2015-2021 SonarSource SA
- * mailto: contact AT sonarsource DOT com
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 3 of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public License
- * along with this program; if not, write to the Free Software Foundation,
- * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
- */
-
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Reflection;
-using System.Text.RegularExpressions;
-using FluentAssertions;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using SonarAnalyzer.Common;
-using SonarAnalyzer.Helpers;
-using SonarAnalyzer.Rules;
-using SonarAnalyzer.Utilities;
-
-namespace SonarAnalyzer.UnitTest.ResourceTests
-{
-    [TestClass]
-    public class GeneratedResourcesTest
```

Comment:
Not sure I understand why this was removed. It does not seem related to this PR.

Reply:
Doesn't seem intentional at all.

Reply:
Ok, so I probably deleted it by accident when moving files out of `ResourcesTest` directory. 
The `ResourcesTest` directory is meant to be a data-only directory, see this
https://github.com/SonarSource/sonar-dotnet/blob/0cd6231a8756eaee401ba9dfcc5b7f4d37165e9d/analyzers/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj#L102-L104

So we should rename it to `TestResources` instead (to have it near `TestCases`) in another PR.

I've moved the file to another directory to decofuse its location.